core/wallet/tron_test.go

```diff
@@ -0,0 +1,58 @@
+package wallet
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	coremock "github.com/TRON-US/go-btfs/core/mock"
+
+	"github.com/tron-us/go-btfs-common/crypto"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetHexAddress(t *testing.T) {
+	privKey, err := crypto.ToPrivKey("CAISIBMlMbrGwtWKz894jM5LGyEMaT5O/FkklTGoF7mQViJj")
+	if err != nil {
+		t.Error(err)
+	}
+	hexAddr, err := getHexAddress(privKey)
+	if err != nil {
+		t.Error(err)
+	}
+	fmt.Println("hexAddr", hexAddr)
+}
+
+func TestTransferBTT(t *testing.T) {
+	//ignore this test
+	t.SkipNow()
```

Comment:
should test using testnet tokens.

Reply:
dev token is mainnet token now.
met some errors when issued a new token in testnet, will try again later.

Reply:
can you not use this dev token here: https://github.com/TRON-US/go-btfs/blob/master/core/wallet/wallet.go#L35

Reply:
i can use another token(testnet or mainnet) later.
but looks like there are something wrong when i issue a new token.
will try again later, i use btt token to work around it.
(i don't have enough 1002508 tokens now.)

Reply:
Oh I see. This was probably just a random token.

Reply:
yes, but exchange uses it in dev/staging....

Reply:
for unit test i can use another random token, it's just not working now for issueing a new token.